.travis.yml

```diff
@@ -64,20 +64,25 @@ install:
       openssl aes-256-cbc -K $encrypted_7e0a8632ac3f_key -iv $encrypted_7e0a8632ac3f_iv -in credentials.tar.gz.enc -out credentials.tar.gz -d ;
       tar -xzf credentials.tar.gz ;
       gcloud auth activate-service-account --key-file credentials/client-secret-docs.json ;
-      gsutil cp gs://simpeg/_build.tar.gz $PWD/docs/ ;
-      mkdir $PWD/docs/_build ;
-      tar -xzf $PWD/docs/_build.tar.gz -C $PWD/docs/_build ;
-      ls $PWD/docs/_build/ ;
+      gsutil cp gs://simpeg/_build.tar.gz $PWD/docs ;
+      gsutil cp gs://simpeg/examples.tar.gz $PWD/docs/content ;
+      cd docs ;
+      tar -xzf _build.tar.gz _build;
+      cd content ;
+      tar -xzf examples.tar.gz examples;
+      cd ../../ ;
     fi
 
 # Run test
 script:
   # - nosetests $TEST_DIR --with-cov --cov SimPEG --cov-config .coveragerc -v -s
   - if [ "$TEST_DIR" = "$DEPLOY_DIR" -a ${TRAVIS_PYTHON_VERSION:0:1} == "3" ]; then
-      cd docs; make html-noplot ; cd ../;
+      cd docs; make html-noplot ; cd ../ ;
```

Comment:
on python 3, we check that the docs compile (no nitpick and no testing of examples)